# Created at 2021-12-30T17:55:38.359
TestSet has not finished before stream error has appeared >> initializing exit by non-null configuration: null
java.io.EOFException
	at java.base/java.io.DataInputStream.readInt(DataInputStream.java:397)
	at org.apache.maven.surefire.booter.MasterProcessCommand.decode(MasterProcessCommand.java:115)
	at org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:391)
	at java.base/java.lang.Thread.run(Thread.java:834)


# Created at 2021-12-30T17:55:39.311
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testGuardar,null,null,null


# Created at 2021-12-30T17:55:39.470
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testGuardar,null,null,null


# Created at 2021-12-30T17:55:39.473
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testDetalle,null,null,null


# Created at 2021-12-30T17:55:39.492
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testDetalle,null,null,null


# Created at 2021-12-30T17:55:39.494
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testListar,null,null,null


# Created at 2021-12-30T17:55:39.520
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testListar,null,null,null


# Created at 2021-12-30T17:55:39.522
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testTransferir,null,null,null


# Created at 2021-12-30T17:55:39.526
Unexpected IOException with stream: {"cuentaOrigenId":1,"cuentaDestinoId":2,"monto":100,"bancoId":1}


# Created at 2021-12-30T17:55:39.535
Unexpected IOException with stream: {"date":"2021-12-30","transaccion":{"cuentaOrigenId":1,"cuentaDestinoId":2,"monto":100,"bancoId":1},"mensaje":"Transferencia realizada con éxito!","status":"OK"}


# Created at 2021-12-30T17:55:39.544
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTest,testTransferir,null,null,null


# Created at 2021-12-30T17:55:39.546
Unexpected IOException: 2,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.controllers.CuentaControllerTest,null,null,null


# Created at 2021-12-30T17:55:39.550
Unexpected IOException: 1,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,null,null,null


# Created at 2021-12-30T17:55:42.476
Unexpected IOException with stream: 2021-12-30 17:55:42.474 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists bancos CASCADE 


# Created at 2021-12-30T17:55:42.478
Unexpected IOException with stream: 2021-12-30 17:55:42.478 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists cuentas CASCADE 


# Created at 2021-12-30T17:55:42.481
Unexpected IOException with stream: 2021-12-30 17:55:42.481 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table bancos (id bigint generated by default as identity, nombre varchar(255), total_transferencias integer, primary key (id))


# Created at 2021-12-30T17:55:42.490
Unexpected IOException with stream: 2021-12-30 17:55:42.489 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table cuentas (id bigint generated by default as identity, persona varchar(255), saldo decimal(19,2), primary key (id))


# Created at 2021-12-30T17:55:42.493
Unexpected IOException with stream: 2021-12-30 17:55:42.493 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('AndrÃ©s', 1000)


# Created at 2021-12-30T17:55:42.496
Unexpected IOException with stream: 2021-12-30 17:55:42.495 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('John', 2000)


# Created at 2021-12-30T17:55:42.497
Unexpected IOException with stream: 2021-12-30 17:55:42.496 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO bancos (nombre, total_transferencias) VALUES ('El banco financiero', 0)


# Created at 2021-12-30T17:55:44.017
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testTransferir,null,null,null


# Created at 2021-12-30T17:55:44.144
Unexpected IOException with stream: 2021-12-30 17:55:44.143 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:44.173
Unexpected IOException with stream: 2021-12-30 17:55:44.173 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:44.175
Unexpected IOException with stream: 2021-12-30 17:55:44.175 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : select banco0_.id as id1_0_0_, banco0_.nombre as nombre2_0_0_, banco0_.total_transferencias as total_tr3_0_0_ from bancos banco0_ where banco0_.id=?


# Created at 2021-12-30T17:55:44.186
Unexpected IOException with stream: 2021-12-30 17:55:44.186 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : update cuentas set persona=?, saldo=? where id=?


# Created at 2021-12-30T17:55:44.190
Unexpected IOException with stream: 2021-12-30 17:55:44.190 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : update cuentas set persona=?, saldo=? where id=?


# Created at 2021-12-30T17:55:44.191
Unexpected IOException with stream: 2021-12-30 17:55:44.191 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : update bancos set nombre=?, total_transferencias=? where id=?


# Created at 2021-12-30T17:55:44.203
Unexpected IOException with stream: 65341

# Created at 2021-12-30T17:55:44.204
Unexpected IOException with stream: 


# Created at 2021-12-30T17:55:44.204
Unexpected IOException with stream: {"date":"2021-12-30","transaccion":{"cuentaOrigenId":1,"cuentaDestinoId":2,"monto":100,"bancoId":1},"mensaje":"Transferencia realizada con éxito!","status":"OK"}


# Created at 2021-12-30T17:55:44.216
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testTransferir,null,null,null


# Created at 2021-12-30T17:55:44.218
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testDetalle,null,null,null


# Created at 2021-12-30T17:55:44.227
Unexpected IOException with stream: 2021-12-30 17:55:44.226 DEBUG 9468 --- [o-auto-1-exec-2] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:44.244
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testDetalle,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,CuentaControllerTestRestTemplateTest.testDetalle:90 expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,org.opentest4j.AssertionFailedError: expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest.testDetalle(CuentaControllerTestRestTemplateTest.java:90)\000A


# Created at 2021-12-30T17:55:44.245
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testListar,null,null,null


# Created at 2021-12-30T17:55:44.274
Unexpected IOException with stream: 2021-12-30 17:55:44.274 DEBUG 9468 --- [o-auto-1-exec-3] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:44.282
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testListar,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,CuentaControllerTestRestTemplateTest.testListar:106 expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,org.opentest4j.AssertionFailedError: expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest.testListar(CuentaControllerTestRestTemplateTest.java:106)\000A


# Created at 2021-12-30T17:55:44.283
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testGuardar,null,null,null


# Created at 2021-12-30T17:55:44.303
Unexpected IOException with stream: 2021-12-30 17:55:44.303 DEBUG 9468 --- [o-auto-1-exec-4] org.hibernate.SQL                        : insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:44.309
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testGuardar,null,null,null


# Created at 2021-12-30T17:55:44.310
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testEliminar,null,null,null


# Created at 2021-12-30T17:55:44.314
Unexpected IOException with stream: 2021-12-30 17:55:44.313 DEBUG 9468 --- [o-auto-1-exec-5] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:44.331
Unexpected IOException with stream: 2021-12-30 17:55:44.330 DEBUG 9468 --- [o-auto-1-exec-6] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:44.337
Unexpected IOException with stream: 2021-12-30 17:55:44.336 DEBUG 9468 --- [o-auto-1-exec-6] org.hibernate.SQL                        : delete from cuentas where id=?


# Created at 2021-12-30T17:55:44.342
Unexpected IOException with stream: 2021-12-30 17:55:44.341 DEBUG 9468 --- [o-auto-1-exec-7] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:44.347
Unexpected IOException with stream: 2021-12-30 17:55:44.347 DEBUG 9468 --- [o-auto-1-exec-8] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:44.352
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,testEliminar,null,null,null


# Created at 2021-12-30T17:55:44.353
Unexpected IOException: 2,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.controllers.CuentaControllerTestRestTemplateTest,null,null,null


# Created at 2021-12-30T17:55:44.354
Unexpected IOException: 1,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,null,null,null


# Created at 2021-12-30T17:55:44.455
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testTransferir,null,null,null


# Created at 2021-12-30T17:55:46.250
Unexpected IOException with stream: 2021-12-30 17:55:46.249 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.252
Unexpected IOException with stream: 2021-12-30 17:55:46.252 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.253
Unexpected IOException with stream: 2021-12-30 17:55:46.253 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : select banco0_.id as id1_0_0_, banco0_.nombre as nombre2_0_0_, banco0_.total_transferencias as total_tr3_0_0_ from bancos banco0_ where banco0_.id=?


# Created at 2021-12-30T17:55:46.255
Unexpected IOException with stream: 2021-12-30 17:55:46.255 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : update cuentas set persona=?, saldo=? where id=?


# Created at 2021-12-30T17:55:46.256
Unexpected IOException with stream: 2021-12-30 17:55:46.256 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : update cuentas set persona=?, saldo=? where id=?


# Created at 2021-12-30T17:55:46.257
Unexpected IOException with stream: 2021-12-30 17:55:46.256 DEBUG 9468 --- [o-auto-1-exec-9] org.hibernate.SQL                        : update bancos set nombre=?, total_transferencias=? where id=?


# Created at 2021-12-30T17:55:46.289
Unexpected IOException: 6,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testTransferir,null,null,null


# Created at 2021-12-30T17:55:46.290
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testDetalle,null,null,null


# Created at 2021-12-30T17:55:46.296
Unexpected IOException with stream: 2021-12-30 17:55:46.296 DEBUG 9468 --- [-auto-1-exec-10] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.303
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testDetalle,null,JSON path "$.persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,null,JSON path "$.persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,CuentaControllerWebTestClientTests.testDetalle:96 JSON path "$.persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,java.lang.AssertionError: JSON path "$.persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testDetalle(CuentaControllerWebTestClientTests.java:96)\000A


# Created at 2021-12-30T17:55:46.305
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testDetalle2,null,null,null


# Created at 2021-12-30T17:55:46.310
Unexpected IOException with stream: 2021-12-30 17:55:46.310 DEBUG 9468 --- [o-auto-1-exec-1] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.321
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testDetalle2,null,expected: <2100.00> but was: <2200.00>,null,expected: <2100.00> but was: <2200.00>,CuentaControllerWebTestClientTests.testDetalle2:109->lambda$testDetalle2$2:113 expected: <2100.00> but was: <2200.00>,org.opentest4j.AssertionFailedError: expected: <2100.00> but was: <2200.00>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.lambda$testDetalle2$2(CuentaControllerWebTestClientTests.java:113)\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testDetalle2(CuentaControllerWebTestClientTests.java:109)\000A


# Created at 2021-12-30T17:55:46.322
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testListar,null,null,null


# Created at 2021-12-30T17:55:46.328
Unexpected IOException with stream: 2021-12-30 17:55:46.327 DEBUG 9468 --- [o-auto-1-exec-2] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.334
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testListar,null,JSON path "$[0].persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,null,JSON path "$[0].persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,CuentaControllerWebTestClientTests.testListar:124 JSON path "$[0].persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>,java.lang.AssertionError: JSON path "$[0].persona" expected:<Andr\00E9s> but was:<Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testListar(CuentaControllerWebTestClientTests.java:124)\000A


# Created at 2021-12-30T17:55:46.335
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testListar2,null,null,null


# Created at 2021-12-30T17:55:46.340
Unexpected IOException with stream: 2021-12-30 17:55:46.339 DEBUG 9468 --- [o-auto-1-exec-3] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.355
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testListar2,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,CuentaControllerWebTestClientTests.testListar2:141->lambda$testListar2$3:146 expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,org.opentest4j.AssertionFailedError: expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.lambda$testListar2$3(CuentaControllerWebTestClientTests.java:146)\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testListar2(CuentaControllerWebTestClientTests.java:141)\000A


# Created at 2021-12-30T17:55:46.356
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testGuardar,null,null,null


# Created at 2021-12-30T17:55:46.361
Unexpected IOException with stream: 2021-12-30 17:55:46.360 DEBUG 9468 --- [o-auto-1-exec-4] org.hibernate.SQL                        : insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.366
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testGuardar,null,JSON path "$.id" expected:<3> but was:<4>,null,JSON path "$.id" expected:<3> but was:<4>,CuentaControllerWebTestClientTests.testGuardar:171 JSON path "$.id" expected:<3> but was:<4>,java.lang.AssertionError: JSON path "$.id" expected:<3> but was:<4>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testGuardar(CuentaControllerWebTestClientTests.java:171)\000A


# Created at 2021-12-30T17:55:46.367
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testGuardar2,null,null,null


# Created at 2021-12-30T17:55:46.373
Unexpected IOException with stream: 2021-12-30 17:55:46.373 DEBUG 9468 --- [o-auto-1-exec-5] org.hibernate.SQL                        : insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.378
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testGuardar2,null,expected: <4> but was: <5>,null,expected: <4> but was: <5>,CuentaControllerWebTestClientTests.testGuardar2:192->lambda$testGuardar2$4:195 expected: <4> but was: <5>,org.opentest4j.AssertionFailedError: expected: <4> but was: <5>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.lambda$testGuardar2$4(CuentaControllerWebTestClientTests.java:195)\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testGuardar2(CuentaControllerWebTestClientTests.java:192)\000A


# Created at 2021-12-30T17:55:46.379
Unexpected IOException: 5,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testEliminar,null,null,null


# Created at 2021-12-30T17:55:46.383
Unexpected IOException with stream: 2021-12-30 17:55:46.383 DEBUG 9468 --- [o-auto-1-exec-6] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.390
Unexpected IOException with stream: 2021-12-30 17:55:46.390 DEBUG 9468 --- [o-auto-1-exec-7] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.407
Unexpected IOException: 8,1,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,testEliminar,null,Status expected:<204 NO_CONTENT> but was:<500 INTERNAL_SERVER_ERROR>,null,Status expected:<204 NO_CONTENT> but was:<500 INTERNAL_SERVER_ERROR>,CuentaControllerWebTestClientTests.testEliminar:212 Status expected:<204 NO_CONTENT> but was:<500 INTERNAL_SERVER_ERROR>,java.lang.AssertionError: Status expected:<204 NO_CONTENT> but was:<500 INTERNAL_SERVER_ERROR>\000A\0009at com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests.testEliminar(CuentaControllerWebTestClientTests.java:212)\000A


# Created at 2021-12-30T17:55:46.408
Unexpected IOException: 2,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.controllers.CuentaControllerWebTestClientTests,null,null,null


# Created at 2021-12-30T17:55:46.409
Unexpected IOException: 1,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.IntegracionJpaTest,null,null,null


# Created at 2021-12-30T17:55:46.699
Unexpected IOException with stream: 2021-12-30 17:55:46.698 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists bancos CASCADE 


# Created at 2021-12-30T17:55:46.699
Unexpected IOException with stream: Hibernate: drop table if exists bancos CASCADE 


# Created at 2021-12-30T17:55:46.700
Unexpected IOException with stream: 2021-12-30 17:55:46.700 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists cuentas CASCADE 


# Created at 2021-12-30T17:55:46.701
Unexpected IOException with stream: Hibernate: drop table if exists cuentas CASCADE 


# Created at 2021-12-30T17:55:46.702
Unexpected IOException with stream: 2021-12-30 17:55:46.701 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table bancos (id bigint generated by default as identity, nombre varchar(255), total_transferencias integer, primary key (id))


# Created at 2021-12-30T17:55:46.702
Unexpected IOException with stream: Hibernate: create table bancos (id bigint generated by default as identity, nombre varchar(255), total_transferencias integer, primary key (id))


# Created at 2021-12-30T17:55:46.705
Unexpected IOException with stream: 2021-12-30 17:55:46.704 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table cuentas (id bigint generated by default as identity, persona varchar(255), saldo decimal(19,2), primary key (id))


# Created at 2021-12-30T17:55:46.705
Unexpected IOException with stream: Hibernate: create table cuentas (id bigint generated by default as identity, persona varchar(255), saldo decimal(19,2), primary key (id))


# Created at 2021-12-30T17:55:46.708
Unexpected IOException with stream: 2021-12-30 17:55:46.707 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('AndrÃ©s', 1000)


# Created at 2021-12-30T17:55:46.709
Unexpected IOException with stream: Hibernate: INSERT INTO cuentas (persona, saldo) VALUES ('AndrÃ©s', 1000)


# Created at 2021-12-30T17:55:46.710
Unexpected IOException with stream: 2021-12-30 17:55:46.709 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('John', 2000)


# Created at 2021-12-30T17:55:46.711
Unexpected IOException with stream: Hibernate: INSERT INTO cuentas (persona, saldo) VALUES ('John', 2000)


# Created at 2021-12-30T17:55:46.712
Unexpected IOException with stream: 2021-12-30 17:55:46.711 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO bancos (nombre, total_transferencias) VALUES ('El banco financiero', 0)


# Created at 2021-12-30T17:55:46.712
Unexpected IOException with stream: Hibernate: INSERT INTO bancos (nombre, total_transferencias) VALUES ('El banco financiero', 0)


# Created at 2021-12-30T17:55:46.818
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindByPersona,null,null,null


# Created at 2021-12-30T17:55:46.838
Unexpected IOException with stream: 2021-12-30 17:55:46.837 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_ where cuenta0_.persona=?


# Created at 2021-12-30T17:55:46.838
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_ where cuenta0_.persona=?


# Created at 2021-12-30T17:55:46.844
Unexpected IOException: 8,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindByPersona,null,expected: <true> but was: <false>,null,expected: <true> but was: <false>,IntegracionJpaTest.testFindByPersona:31 expected: <true> but was: <false>,org.opentest4j.AssertionFailedError: expected: <true> but was: <false>\000A\0009at com.oscar.test.springboot.app.IntegracionJpaTest.testFindByPersona(IntegracionJpaTest.java:31)\000A


# Created at 2021-12-30T17:55:46.845
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindAll,null,null,null


# Created at 2021-12-30T17:55:46.850
Unexpected IOException with stream: 2021-12-30 17:55:46.850 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.851
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.855
Unexpected IOException: 6,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindAll,null,null,null


# Created at 2021-12-30T17:55:46.857
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testSave,null,null,null


# Created at 2021-12-30T17:55:46.861
Unexpected IOException with stream: 2021-12-30 17:55:46.861 DEBUG 9468 --- [           main] org.hibernate.SQL                        : insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.862
Unexpected IOException with stream: Hibernate: insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.866
Unexpected IOException: 6,1,com.oscar.test.springboot.app.IntegracionJpaTest,testSave,null,null,null


# Created at 2021-12-30T17:55:46.868
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindById,null,null,null


# Created at 2021-12-30T17:55:46.872
Unexpected IOException with stream: 2021-12-30 17:55:46.872 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.873
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.879
Unexpected IOException: 8,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindById,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,null,expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,IntegracionJpaTest.testFindById:25 expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>,org.opentest4j.AssertionFailedError: expected: <Andr\00E9s> but was: <Andr\00C3\00A9s>\000A\0009at com.oscar.test.springboot.app.IntegracionJpaTest.testFindById(IntegracionJpaTest.java:25)\000A


# Created at 2021-12-30T17:55:46.880
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testDelete,null,null,null


# Created at 2021-12-30T17:55:46.883
Unexpected IOException with stream: 2021-12-30 17:55:46.883 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.884
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_0_, cuenta0_.persona as persona2_1_0_, cuenta0_.saldo as saldo3_1_0_ from cuentas cuenta0_ where cuenta0_.id=?


# Created at 2021-12-30T17:55:46.888
Unexpected IOException with stream: 2021-12-30 17:55:46.888 DEBUG 9468 --- [           main] org.hibernate.SQL                        : delete from cuentas where id=?


# Created at 2021-12-30T17:55:46.889
Unexpected IOException with stream: Hibernate: delete from cuentas where id=?


# Created at 2021-12-30T17:55:46.890
Unexpected IOException with stream: 2021-12-30 17:55:46.889 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.890
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_


# Created at 2021-12-30T17:55:46.893
Unexpected IOException: 6,1,com.oscar.test.springboot.app.IntegracionJpaTest,testDelete,null,null,null


# Created at 2021-12-30T17:55:46.894
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindByPersonaThrowException,null,null,null


# Created at 2021-12-30T17:55:46.897
Unexpected IOException with stream: 2021-12-30 17:55:46.896 DEBUG 9468 --- [           main] org.hibernate.SQL                        : select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_ where cuenta0_.persona=?


# Created at 2021-12-30T17:55:46.897
Unexpected IOException with stream: Hibernate: select cuenta0_.id as id1_1_, cuenta0_.persona as persona2_1_, cuenta0_.saldo as saldo3_1_ from cuentas cuenta0_ where cuenta0_.persona=?


# Created at 2021-12-30T17:55:46.900
Unexpected IOException: 6,1,com.oscar.test.springboot.app.IntegracionJpaTest,testFindByPersonaThrowException,null,null,null


# Created at 2021-12-30T17:55:46.901
Unexpected IOException: 5,1,com.oscar.test.springboot.app.IntegracionJpaTest,testUpdate,null,null,null


# Created at 2021-12-30T17:55:46.904
Unexpected IOException with stream: 2021-12-30 17:55:46.904 DEBUG 9468 --- [           main] org.hibernate.SQL                        : insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.905
Unexpected IOException with stream: Hibernate: insert into cuentas (id, persona, saldo) values (null, ?, ?)


# Created at 2021-12-30T17:55:46.908
Unexpected IOException: 6,1,com.oscar.test.springboot.app.IntegracionJpaTest,testUpdate,null,null,null


# Created at 2021-12-30T17:55:46.909
Unexpected IOException: 2,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.IntegracionJpaTest,null,null,null


# Created at 2021-12-30T17:55:46.910
Unexpected IOException: 1,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.SpringbootTestApplicationTests,null,null,null


# Created at 2021-12-30T17:55:47.420
Unexpected IOException with stream: 2021-12-30 17:55:47.419 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists bancos CASCADE 


# Created at 2021-12-30T17:55:47.421
Unexpected IOException with stream: 2021-12-30 17:55:47.420 DEBUG 9468 --- [           main] org.hibernate.SQL                        : drop table if exists cuentas CASCADE 


# Created at 2021-12-30T17:55:47.421
Unexpected IOException with stream: 2021-12-30 17:55:47.421 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table bancos (id bigint generated by default as identity, nombre varchar(255), total_transferencias integer, primary key (id))


# Created at 2021-12-30T17:55:47.424
Unexpected IOException with stream: 2021-12-30 17:55:47.423 DEBUG 9468 --- [           main] org.hibernate.SQL                        : create table cuentas (id bigint generated by default as identity, persona varchar(255), saldo decimal(19,2), primary key (id))


# Created at 2021-12-30T17:55:47.427
Unexpected IOException with stream: 2021-12-30 17:55:47.426 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('AndrÃ©s', 1000)


# Created at 2021-12-30T17:55:47.428
Unexpected IOException with stream: 2021-12-30 17:55:47.428 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO cuentas (persona, saldo) VALUES ('John', 2000)


# Created at 2021-12-30T17:55:47.429
Unexpected IOException with stream: 2021-12-30 17:55:47.428 DEBUG 9468 --- [           main] org.hibernate.SQL                        : INSERT INTO bancos (nombre, total_transferencias) VALUES ('El banco financiero', 0)


# Created at 2021-12-30T17:55:48.177
Unexpected IOException: 5,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,testFindAll,null,null,null


# Created at 2021-12-30T17:55:48.183
Unexpected IOException: 6,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,testFindAll,null,null,null


# Created at 2021-12-30T17:55:48.185
Unexpected IOException: 5,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,testSave,null,null,null


# Created at 2021-12-30T17:55:48.190
Unexpected IOException: 6,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,testSave,null,null,null


# Created at 2021-12-30T17:55:48.191
Unexpected IOException: 5,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads2,null,null,null


# Created at 2021-12-30T17:55:48.200
Unexpected IOException: 6,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads2,null,null,null


# Created at 2021-12-30T17:55:48.201
Unexpected IOException: 5,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads3,null,null,null


# Created at 2021-12-30T17:55:48.207
Unexpected IOException: 6,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads3,null,null,null


# Created at 2021-12-30T17:55:48.208
Unexpected IOException: 5,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads,null,null,null


# Created at 2021-12-30T17:55:48.216
Unexpected IOException: 6,1,com.oscar.test.springboot.app.SpringbootTestApplicationTests,contextLoads,null,null,null


# Created at 2021-12-30T17:55:48.217
Unexpected IOException: 2,1,org.apache.maven.surefire.junitplatform.JUnitPlatformProvider,com.oscar.test.springboot.app.SpringbootTestApplicationTests,null,null,null


